woocommerce/woocommerce-android
# Feature/9963 campaign list pagination

Closes: #9994


### Description

Adds pagination logic for fetching campaign list.

### Testing instructions

Testing this is not straightforward at all as there is no way to create Blaze test campaigns in bulk. The page size for the API is 10 elements, so you'll need to manually create at least 11 campaigns to test pagination. What I did is use Proxyman to override the responses to return different pages of campaign items. If you already have Proxyman or other tool for overriding responses, feel free to test this, if not, you can trust that I tested it and see the demo: 


https://github.com/woocommerce/woocommerce-android/assets/2663464/d8bc9ad6-96ec-4fb8-b147-16192208bbf7
